scripts/copy_output.lua

```diff
@@ -53,6 +54,20 @@ for _, image in ipairs(lib.images) do
 	end
 end
 
+-- copy kernel image with debug symbols
+local kernel_debug_glob = string.format('%s/debug/\0-%s-kernel-debug.tar.bz2',
+        env.GLUON_OUTPUTDIR,
+        target)
```

Comment:
The `mkdir` creates `${GLUON_IMAGEDIR}/debug`, but this attempts to write to `${GLUON_OUTPUTDIR}/debug`.

Reply:
Either use `${GLUON_IMAGEDIR}` here, or define a separate variable `${GLUON_DEBUGDIR}` in the main Makefile to make this configurable, similar to `${GLUON_PACKAGEDIR}`.

Reply:
> Build stage fails:
> 
> ```
> cp: cannot create regular file '/home/jenkins/workspace/gluon-ci_PR-1971/output/debug/ci-0.6+exp20200422-x86-64-kernel-debug.tar.bz2': No such file or directory
> openwrt/staging_dir/hostpkg/bin/lua: scripts/target_lib.lua:114: assertion failed!
> stack traceback:
> 	[C]: in function 'assert'
> 	scripts/target_lib.lua:114: in function <scripts/target_lib.lua:112>
> 	(tail call): ?
> 	scripts/copy_output.lua:69: in main chunk
> 	[C]: ?
> make: *** [Makefile:196: all] Error 1
> ```
This should not have happend. I guess my test environment wasn't clean enough.

I now have defined `${GLUON_DEBUGDIR}` in the Makefile and have exported it into the environment. And I am now using it to create the directory and to copy files there...
One can now override it like `${GLUON_IMAGEDIR}`.